cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -229,6 +229,8 @@ spec:
 #      config: # configuration can be specified at the node level which overrides the cluster level config
 #    - name: "172.17.4.301"
 #      deviceFilter: "^sd."
+    # when onlyApplySpecificOSDPlacement false, will merge both placement.All() and storageClassDeviceSets.Placement.
```

Comment:
same comment `when onlyApplySpecificOSDPlacement false, will merge both placement.All() and storageClassDeviceSets.Placement.` is used for both cluster.yaml and cluster-on-pvc.yaml.   I think we need to differentiate between osd-on-pvc and osd-on-disks scenarios here.

Reply:
updated.